deepspeed/runtime/engine.py

```diff
@@ -1059,6 +1059,9 @@ def zeropp_loco_param(self):
     def zero_log_trace_cache_warnings(self):
         return self._config.zero_config.log_trace_cache_warnings
 
+    def enable_sanity_checks(self):
+        return self._config.zero_config.enable_sanity_checks
```

Comment:
this feels misnamed, since it doesn't enable anything, perhaps:

```suggestion
    def is_sanity_checks_enabled(self):
        return self._config.zero_config.enable_sanity_checks
```